src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -1279,9 +1286,13 @@ private void startSyncWithPeer(PeerConnection peer) {
     syncNextBatchChainIds(peer);
   }
 
-  private void syncNextBatchChainIds(PeerConnection peer) {
+  synchronized private void syncNextBatchChainIds(PeerConnection peer) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reorder the modifiers to comply with the Java Language Specification. ([squid:ModifiersOrderCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AModifiersOrderCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1672&issues=AWbTwVAGO08i52vd_Pwk&open=AWbTwVAGO08i52vd_Pwk)

[ISSUE_KEY]: AWbTwVAGO08i52vd_Pwk